docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+# Declarative peering
+
+Declarative peerings are supported starting from Liqo 1.0, which means that you can create a set of CRs describing the peering with a remote cluster, which is automatically set up once the CRs are applied on both sides.
+This simplifies automation, GitOps and continuous delivery: for example, you might have your Git repository with the manifests describing the peerings, and an instance of [ArgoCD](https://argo-cd.readthedocs.io) which synchronizes the changes on the clusters, creating and destroying the peerings.
+
+The following lines analizes how to describe the configuration of each of the Liqo modules:
+
+- Networking
+- Authentication
+- Offloading
+
+## Tenant namespace
+
+Before starting to configure the peerings, you need to create a tenant namespace in both clusters that you need to peer.
+This tenant namespace must **refer to the peering with a specific cluster**, hence a distinct tenant namespace must be created per each peering.
+**All the resources needed to configure the peering must be created in those namespaces**.
+A tenant namespace **can have an arbitrary name**, but **it must have the following labels**:
+
+```text
+liqo.io/remote-cluster-id: <HERE_THE_CLUSTER_ID_OF_PEER_CLUSTER>
+liqo.io/tenant-namespace: "true"
+```
+
+### Configuring the tenant namespace on cluster consumer
+
+The following is an example of tenant namespace referring to the peering with a cluster with id `cl-provider`:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: cl-provider
+    liqo.io/tenant-namespace: "true"
+  name: liqo-tenant-cl-provider
+spec: {}
+```
+
+### Configuring the tenant namespace on the cluster provider
+
+**You will need a similar namespace on the provider cluster** pointing to the cluster ID of your local cluster (cluster consumer).
```

Comment:
I would shorten this text, as in the previous subsection.
I would also add that the namespaces on the two clusters do not need to have the same name.